components/atoms/Search/search.tsx

```diff
@@ -150,10 +149,19 @@ const Search = ({
                 style={suggestionsStyle}
                 key={index}
                 data-suggestion={typeof suggestion === "string" ? suggestion : suggestion.key}
+                role="button"
```

Comment:
This should be an actual button, and we can remove the `tabIndex`, `onKeyDown` and `role="button"`